subiquitycore/pubsub.py

```diff
@@ -36,5 +36,5 @@ async def abroadcast(self, channel, *args, **kwargs):
                 await v
 
     def broadcast(self, channel, *args, **kwargs):
-        loop = asyncio.get_event_loop()
+        loop = asyncio.get_running_loop()
```

Comment:
This can just be asyncio.create_task I think.

Reply:
Good catch, thanks!